### PR TITLE
[xc-admin] delete product

### PR DESCRIPTION
**Pull request by cctdaniel**

### CODE REVIEW

File: governance/xc_admin/packages/xc_admin_frontend/components/tabs/General.tsx [hunk 1]
```diff
@@ -197,6 +197,17 @@ const General = () => {
               changes[symbol].new = { ...fileDataParsed[symbol] }
             }
           })
+          // check if any existing symbols are not in uploaded json
+          Object.keys(data).forEach((symbol) => {
+            if (!fileDataParsed[symbol]) {
+              changes[symbol] = { prev: {} }
+              changes[symbol].prev = { ...data[symbol] }
+              changes[symbol].prev.metadata = {
```
Comment (guibescos) [inline governance/xc_admin/packages/xc_admin_frontend/components/tabs/General.tsx:205]: This line doesn't do anything I think

Comment (cctdaniel) [reply governance/xc_admin/packages/xc_admin_frontend/components/tabs/General.tsx:205]: what do you mean? `prev.metadata` doesn't have symbol, this line adds symbol to metadata

Comment (guibescos) [reply governance/xc_admin/packages/xc_admin_frontend/components/tabs/General.tsx:205]: You'll never use symbol for a product you're about to delete

**Review by guibescos — APPROVED** [review]

File: governance/xc_admin/packages/xc_admin_frontend/components/tabs/General.tsx (hunk 1 above)
Comment (cctdaniel) [reply governance/xc_admin/packages/xc_admin_frontend/components/tabs/General.tsx:205]: ah yeah you're right

**Pull request merged**